lib/screens_web/views/v2/audio/departures_view.ex

```diff
@@ -59,7 +59,8 @@ defmodule ScreensWeb.V2.Audio.DeparturesView do
     ~E|<s><%= content %></s>|
   end
 
-  defp render_time_with_crowding({%{crowding: crowding, time: time}, _}, route, _headsign) do
+  defp render_time_with_crowding({%{crowding: crowding, time: time}, _}, route, headsign) do
+    route_headsign_rendered = render_route_headsign(route, headsign)
```

Comment:
Looks like we only wanted the route + vehicle type for subsequent times, not the full route + vehicle + headsign.
E.g. "A later 22 bus arrives ...", not "A later 22 bus to Jackson Square arrives ..."

This list should give that result when passed to `build_text/1`:
```ex
[
  prefix,
  {route.route_text, &render_route/1},
  route.vehicle_type || "trip",
  ...rest of the existing items
]
```
---

I also noticed a small bug/inconsistency in my own code—in `render_route_headsign/2` could you change this line: `{vehicle_type, fn v -> ~E|<%= v %>| end}` to just `vehicle_type || "trip"`?

Reply:
Good catch. Was so busy listening for the route that I didn't notice the headsign.

Also, on the screen I'm testing on (1401), I noticed that Polly is saying S-T instead of street. Are we ok with this or should we add some logic to get Polly to say the full word?

Reply:
Thanks for catching that! I added a lexicon entry to fix it. (For the record, the instance was "Magazine St @ Green St")